src/Microsoft.HttpRepl/Commands/BaseHttpCommand.cs

```diff
@@ -165,8 +165,21 @@ protected override async Task ExecuteAsync(IShellState shellState, HttpState pro
                 string headersTarget = responseHeadersFileOption?.Text ?? responseFileOption?.Text;
                 string bodyTarget = responseBodyFileOption?.Text ?? responseFileOption?.Text;
 
-                HttpResponseMessage response = await programState.Client.SendAsync(request, HttpCompletionOption.ResponseHeadersRead, cancellationToken).ConfigureAwait(false);
-                await HandleResponseAsync(programState, commandInput, shellState.ConsoleManager, response, programState.EchoRequest, headersTarget, bodyTarget, cancellationToken).ConfigureAwait(false);
```

Comment:
Just curious, if nothing was catching exceptions here, what thing did catch them to allow the exception stack to show in the console?

Reply:
Further up the command execution pipeline:
https://github.com/dotnet/HttpRepl/blob/feb5c12bc9c2d6d72667826d1a5fb2f36ab73a87/src/Microsoft.Repl/Commanding/DefaultCommandDispatcher.cs#L119-L122